package/network/services/dropbear/Makefile

```diff
@@ -8,25 +8,41 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dropbear
-PKG_VERSION:=2022.82
-PKG_RELEASE:=$(AUTORELEASE)
+PKG_VERSION:=2022.83
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:= \
 	https://matt.ucc.asn.au/dropbear/releases/ \
 	https://dropbear.nl/mirror/releases/
-PKG_HASH:=3a038d2bbc02bf28bbdd20c012091f741a3ec5cbe460691811d714876aad75d1
+PKG_HASH:=bc5a121ffbc94b5171ad5ebe01be42746d50aa797c9549a4639894a16749443b
 
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=LICENSE libtomcrypt/LICENSE libtommath/LICENSE
 PKG_CPE_ID:=cpe:/a:matt_johnston:dropbear_ssh_server
 
 PKG_BUILD_PARALLEL:=1
-PKG_ASLR_PIE_REGULAR:=1
 PKG_USE_MIPS16:=0
 PKG_FIXUP:=autoreconf
 PKG_FLAGS:=nonshared
 
+# global hardening fixups
+CONFIG_PKG_ASLR_PIE_REGULAR:=1
+CONFIG_PKG_CC_STACKPROTECTOR_STRONG:=1
+CONFIG_PKG_RELRO_FULL:=1
+CONFIG_PKG_FORTIFY_SOURCE_2:=1
+undefine CONFIG_PKG_ASLR_PIE_ALL
+undefine CONFIG_PKG_CC_STACKPROTECTOR_REGULAR
+undefine CONFIG_PKG_RELRO_PARTIAL
+undefine CONFIG_PKG_FORTIFY_SOURCE_1
+# package hardening fixups
+PKG_ASLR_PIE_REGULAR:=1
+PKG_SSP:=1
+PKG_RELRO:=1
+PKG_FORTIFY_SOURCE:=1
```

Comment:
The 12 lines above  mostly  matches the OpenWrt default configuration already. I think you should not change it here.